src/app/model/migrations/migrations/to0002/MigrationStorageTypes0002.ts

```diff
@@ -0,0 +1,21 @@
+type FlowPipelineItemFrom = {
+  type: 'Stepper' | 'Intermediate' | 'Summary' | 'Finish';
+  payload: {
+    appletId: string;
+    activityId: string;
+    eventId: string;
+    flowId?: string;
+    order: number;
+    activityName: string;
+    activityDescription?: string;
+    activityImage?: string | null;
+  };
+};
+
+export type FlowStateFrom = {
```

Comment:
Please rename the `FlowStateFrom` and `FlowPipelineItemFrom` to `FlowState` and `FlowPipelineItem` as there is no "To" version of the types.

Reply:
Please, don't hurry. Why we have From, but do not have To? 
If you do not migrate it then Flow also should not exist. 
I've noticed that some types defined in the utils?